src/extension/options-page/DashboardComponents/AbstractTab.tsx

```diff
@@ -28,12 +28,14 @@ export interface AbstractTabProps {
 export default function AbstractTab({ tabs, state, height = 200 }: AbstractTabProps) {
     const classes = useStyles()
     const [value, setValue] = state
+    const tabIndicatorStyle = tabs.length > 0 ? { width: 100 / tabs.length + '%' } : {}
```

Comment:
```suggestion
    const tabIndicatorStyle = tabs.length ? { width: (100 / tabs.length) + '%' } : undefined
```

Reply:
done, that bracket was removed by prettier.